File: GARI_Umbria_2026-08-03/GiasNG/src/app/Model/attivita/risorse/RisorsaSpecie.ts
import { Risorsa } from "./Risorsa";
import { Specie } from "app/Model/metaschema/utilizzi/Specie";

export class RisorsaSpecie extends Risorsa {

    specie: Specie;

    constructor() {
        super();
        this.classType = 'RisorsaSpecie';
    }

}

File: GARI_Umbria_2026-08-03/DotNet/AgronicaCoreStd/AgronicaCoreModelsSTD/attivita/risorse/RisorsaSpecie.cs
using AgronicaCoreModelsSTD.metaschema.utilizzi;

namespace AgronicaCoreModelsSTD.attivita.risorse
{
    public class RisorsaSpecie: Risorsa
    {
        public Specie specie { get; set; }

        public RisorsaSpecie()
        {
            classType = costanti.ClassType.RisorsaSpecie;
        }

    }
}
